Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj

```diff
@@ -80,14 +80,17 @@
       * Introduce PH2102: Xml documentation should add value.
       * Fixed issue 156: Multiple false positive on PH2101: Using 'as' expression
       * Improve PH2077: AvoidRedundantSwitch should exclude generated code
+      1.2.33
+      * Introduce PH2103: Avoid method calls as arguments
+      * Introduce PH2104: Every Linq statement on separate line
+      * Introduce PH2105: Reduce Cognitive Load
     </PackageReleaseNotes>
-	  <Copyright>© 2019-2022 Koninklijke Philips N.V.</Copyright>
+	  <Copyright>© 2019-2023 Koninklijke Philips N.V.</Copyright>
 	  <PackageTags>CSharp Maintainability Roslyn CodeAnalysis analyzers Philips</PackageTags>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <DevelopmentDependency>true</DevelopmentDependency>
-    <Version>1.2.32</Version>
+    <Version>1.2.32.4</Version>
```

Comment:
4 version fields ??

Reply:
Oops that's just my local testing. Will fix.